test/sheldon_meta_SUITE.erl

```diff
@@ -0,0 +1,17 @@
+-module(sheldon_meta_SUITE).
+
+-include_lib("mixer/include/mixer.hrl").
+-mixin([{ktn_meta_SUITE
+        , [ all/0
+           , xref/1
```

Comment:
Commas must be aligned with the opening and closing square brackets, something like:

```
[ some_text_here
, another_text_here
, ...
].
```